### ISSUE TITLE
router is not stripping/building paths correctly

### ISSUE THREAD

**Issue by pokemane** [getsentry/sentry#4339]
Server Version
8.8.0
Python Version
2.7.12 (default, Sep 23 2016, 22:25:48) 
[GCC 4.9.2]
Host OS is Windows, 7, using Docker Toolbox.

Installed using Sentry container from the Docker Hub, using the official Sentry repository and the instructions contained there.

Per title.  I have found that some links, when you try to navigate away from them, will not strip their own path before trying to build/append the new one, resulting in something like:

```
http://host/organizations/my-org/api-keys/organizations/my-org/teams/
```

when trying to navigate from the API Keys page to the Projects and Teams page.  I saw that there was an issue like this with react-router that has since been fixed, so I'm not sure what the issue would be.  It seemingly chose "random" pages to do this on, but they're consistent and it even affects navigating to other pages sharing the same parent route, e.g. `/organizations/my-org/api-keys` to `/organizations/my-org/audit-log/` resulting in `/organizations/my-org/api-keys/organizations/my-org/audit-log/`.  I've been setting this up on my local machine to evaluate/test it so I don't have it running behind a web server or with its own domain, though I don't know how that would affect it.

Any ideas?

**Comment by dcramer** [comment]
I haven't ever seen this in our installations, so my gut would tell me this is a misconfiguration, though I don't understand how it'd be inconsistent if that were true.

**Comment by pokemane** [comment]
That was my first reaction too; I was just like 95% sure I had followed the instructions to the letter.  I'll give it a wipe and try again.

**Comment by monjerry** [comment]
I'm experiencing the same, I also use docker. I did make a change in the config but I don't see any harm in it. 

At first the root url in admin/settings was setup with the ip and the port, looking something like this 10.8.8.8:8080

Then I decided to use hostname instead of IP so I changed it to hostname.com:8080

Then I decided to expose the port 80 on docker instead of 8080 so that I could make requests to the sentry machine without specifying any port so I changed it to hostname.com

After this it was that I started seeing that the links error as @pokemane says. Not sure if the links were broken before though
